ecs-cli/modules/compose/ecs/utils/convert_task_definition_test.go

```diff
@@ -387,18 +406,23 @@ func TestConvertToMountPoints(t *testing.T) {
 	}
 }
 
-func verifyMountPoint(t *testing.T, output *ecs.MountPoint, volumes map[string]string,
+func verifyMountPoint(t *testing.T, output *ecs.MountPoint, volumes *volumes,
 	hostPath, containerPath string, readonly bool) {
-
+	sourceVolume := ""
 	if containerPath != *output.ContainerPath {
 		t.Errorf("Expected containerPath [%s] But was [%s]", containerPath, *output.ContainerPath)
 	}
-	sourceVolume := volumes[hostPath]
+	if hostPath != "" {
+		sourceVolume = volumes.volumeHasHost[hostPath]
+	} else {
+		sourceVolume = volumes.volumeEmptyHost[emptyHostCtr]
+		emptyHostCtr++
+	}
 	if sourceVolume != *output.SourceVolume {
 		t.Errorf("Expected sourceVolume [%s] But was [%s]", sourceVolume, *output.SourceVolume)
 	}
 	if readonly != *output.ReadOnly {
-		t.Errorf("Expected readonly [%s] But was [%s]", readonly, *output.ReadOnly)
+		t.Errorf("Expected readonly [%v] But was [%v]", readonly, *output.ReadOnly)
```

Comment:
Adding testify/assert will be a separate PR since it's not specific to this issue and isn't currently used in the file.